proxy/http2/HPACK.h

```diff
@@ -90,31 +97,29 @@ class MIMEFieldWrapper
     return _field;
   }
 
+  bool
+  is_sensitive() const
+  {
+    return _field->is_sensitive();
+  }
+
 private:
   MIMEField *_field;
   HdrHeap *_heap;
   MIMEHdrImpl *_mh;
 };
 
-// Result of looking for a header field in IndexingTable
-struct Http2LookupIndexResult {
-  Http2LookupIndexResult() : index(0), value_is_indexed(false) {}
-
-  int index;
-  bool value_is_indexed;
-};
-
 // [RFC 7541] 2.3.2. Dynamic Table
-class Http2DynamicTable
+class HpackDynamicTable
 {
 public:
-  Http2DynamicTable() : _current_size(0), _settings_dynamic_table_size(4096)
+  HpackDynamicTable() : _current_size(0), _maximum_size(4096)
```

Comment:
The value of 4096 comes from HTTP2, so I changed to pass HTTP2_HEADER_TABLE_SIZE on to the constructor.